src/backend/expungeservice/form_filling.py

```diff
@@ -236,17 +237,17 @@ def _build_pdf_for_eligible_case(
             "case_name": case.summary.name,
             "da_number": case.summary.district_attorney_number,
             "sid": sid,
-            "has_conviction": "✓" if has_conviction else "",
-            "has_no_complaint": "✓" if has_no_complaint else "",
-            "has_dismissed": "✓" if has_dismissals else "",
-            "has_contempt_of_court": "✓" if has_contempt_of_court else "",
+            "has_conviction": "X" if has_conviction else "",
+            "has_no_complaint": "X" if has_no_complaint else "",
+            "has_dismissed": "X" if has_dismissals else "",
+            "has_contempt_of_court": "X" if has_contempt_of_court else "",
             "conviction_dates": "; ".join(conviction_dates),
-            "has_class_b_felony": "✓" if has_class_b_felony else "",
-            "has_class_c_felony": "✓" if has_class_c_felony else "",
-            "has_class_a_misdemeanor": "✓" if has_class_a_misdemeanor else "",
-            "has_class_bc_misdemeanor": "✓" if has_class_bc_misdemeanor else "",
-            "has_violation_or_contempt_of_court": "✓" if has_violation_or_contempt_of_court else "",
-            "has_probation_revoked": "✓" if has_probation_revoked else "",
+            "has_class_b_felony": "X" if has_class_b_felony else "",
+            "has_class_c_felony": "X" if has_class_c_felony else "",
+            "has_class_a_misdemeanor": "X" if has_class_a_misdemeanor else "",
+            "has_class_bc_misdemeanor": "X" if has_class_bc_misdemeanor else "",
+            "has_violation_or_contempt_of_court": "X" if has_violation_or_contempt_of_court else "",
+            "has_probation_revoked": "X" if has_probation_revoked else "",
```

Comment:
Did you check that the offsets work well for this.

Reply:
Here's how it looks with the "oregon_with_conviction.pdf"
<img width="759" alt="image" src="https://user-images.githubusercontent.com/120387970/221384407-ded7a588-0ace-47b3-8702-263c94279fdd.png">

[COMMON NAME_110000_baker_oregon_with_conviction.pdf](https://github.com/codeforpdx/recordexpungPDX/files/10832510/COMMON.NAME_110000_baker_oregon_with_conviction.pdf)

Reply:
Yeah are you able to create the box offset a bit? I had this issue too and offseting the box was a nice hack.